smarts/sstudio/genhistories.py

```diff
@@ -85,7 +85,9 @@ def check_dataset_spec(self, dataset_spec: Dict[str, Any]):
         if "input_path" not in dataset_spec:
             errmsg = "'input_path' field is required in dataset yaml."
         elif dataset_spec.get("flip_y"):
+            # pytype: disable=attribute-error
             if not dataset_spec.get("source").startswith("NGSIM"):
+            # pytype: enable=attribute-error
```

Comment:
Default value of `get()` is `None`. `None` has no `startswith()` attribute.  Hence, consider the following change.
```suggestion
            if not dataset_spec["source"].startswith("NGSIM"):
```